MIGRATE.md

```diff
@@ -105,7 +105,7 @@ $ webpack-cli migrate ./webpack.config.js
 +              })
 ? Are you sure these changes are fine? Yes
 
- ✔︎ New webpack v2 config file is at /Users/obuckley/Workspace/github/repos/webpack-migrate-sandbox/webpack.config.js
+ ✔︎ The new webpack v2 config file is at /Users/obuckley/Workspace/github/repos/webpack-migrate-sandbox/webpack.config.js
```

Comment:
That is the output from `@webpack-cli/migrate`. Revert this change
```suggestion
 ✔︎ New webpack v2 config file is at /Users/obuckley/Workspace/github/repos/webpack-migrate-sandbox/webpack.config.js
```

Reply:
Sure @rishabh3112  will do that